torch/_inductor/__init__.py

```diff
@@ -44,8 +48,8 @@ def compile(
 
 def aoti_compile_and_package(
     exported_program,
-    args: Tuple[Any],
-    kwargs: Optional[Dict[str, Any]] = None,
+    _deprecated_unused_args=None,
+    _deprecated_unused_kwargs=None,
```

Comment:
These were not announced in 2.5, so I feel it is ok to simply drop them. You can do that as a follow-up PR and keep your warnings run for a while for now, as long as the change lands before the 2.6 branch cut date.  There are some internal references need to be updated when you drop these parameters.